services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedTenantService.java

```diff
@@ -106,6 +102,11 @@ public void setConfig(final MongoDbBasedTenantsConfigProperties config) {
 
     @Override
     public void start(final Promise<Void> startPromise) {
+        final String FIELD_PAYLOAD_TRUSTED_CA_WITH_SUBJECT_DN =
```

Comment:
Since this variable is being used at only one place, could you please inline that reference and remove this variable.

Reply:
done